activiti-cloud-services-runtime-bundle/activiti-cloud-services-api/src/main/java/org/activiti/cloud/services/api/commands/CreateTaskCmd.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.api.commands;
+
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CreateTaskCmd implements Command {
```

Comment:
@lucianoprea I do understand that this is the first iteration, but we will need at least an ApplicationName. 
Tasks has a lot of properties which are really important, such as priority and dueDate. What are we using category for?

Reply:
I'm not sure I get the part with **ApplicationName**, can you give more details why we need this? The current behaviour in Activiti6 and APS is that you send a request **POST** _/tasks_ with a body that has this 3 properties and the standalone task is created then if you want to add more data to the task  an update request **PUT** _/tasks/{taskID}_ will add the _dueDate_, _priority_ and other stuff.

Reply:
> What are we using category for?
Found out that category is used to set the id of the application to which an standalone task is part of.

Reply:
If it's used to set the application id then can we rename this field to 'applicationName' and put a comment saying that  it is intended to be similar to usage of the previous category field?

Reply:
I do think we should expose options to set dueDate, priority and probably assignee but I think it would be ok to merge this first and then create a new issue and open a subsequent PR.